scripts/dev.sh

```diff
@@ -131,7 +131,7 @@ else
 fi
 
 tmux new-window -n 'fce' "export ROS_NAMESPACE=$NAME; cd $OCBHAVE/face_track; ./main.py; $SHELL"
-
+tmux new-window -n 'sound' "export ROS_NAMESPACE=$NAME; cd $OCBHAVE/sound; ./audioclass.py; $SHELL"
```

Comment:
I still don't know where this script is. Is it not checked in? This should be checked in first. I'll wait for the code on ros-behavior-scripting to be checked in.